samples/snippets/cloud-client/src/test/java/app/SnippetsIT.java

```diff
@@ -219,33 +223,48 @@ public void testCreateAnnotateAccountDefender()
                 "Finished searching related account group memberships for %s", hashedAccountId));
   }
 
+  @Test
   public void testGetMetrics() throws IOException {
     GetMetrics.getMetrics(PROJECT_ID, RECAPTCHA_SITE_KEY_1);
     assertThat(stdOut.toString())
         .contains("Retrieved the bucket count for score based key: " + RECAPTCHA_SITE_KEY_1);
   }
 
-  public JSONObject createAssessment(String testURL)
-      throws IOException, JSONException, InterruptedException {
+  @Test
+  public void testPasswordLeakAssessment()
+      throws JSONException, IOException, ExecutionException, InterruptedException {
+    String testURL = "http://localhost:" + randomServerPort + "/";
+    createAssessment(testURL, ByteString.EMPTY, AssessmentType.PASSWORD_LEAK);
+    assertThat(stdOut.toString()).contains("Is Credential leaked: ");
+  }
+
+  public JSONObject createAssessment(
```

Comment:
This is ok for now.  In the future it might be a better pattern to create a method for each assessmentType and then call a common method for completion.  It's a bit less error prone and easier to read & understand than if .. else if ... else if ... etc.

Reply:
I've refactored the statement a bit. Will modify it to include a common method like you suggested. Thanks Les!

Reply:
Done. Thanks.